packages/fast-components-styles-msft/src/flipper/index.ts

```diff
@@ -0,0 +1,68 @@
+import designSystemDefaults, { IDesignSystem } from "../design-system";
+import { ComponentStyles, ComponentStyleSheet } from "@microsoft/fast-jss-manager";
+import { IFlipperClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import { applyLocalizedProperty, Direction, toPx } from "@microsoft/fast-jss-utilities";
+import { applyMixedColor } from "../utilities/colors";
+import { get } from "lodash-es";
+import * as Chroma from "chroma-js";
+
+/* tslint:disable:max-line-length */
+const styles: ComponentStyles<IFlipperClassNameContract, IDesignSystem> = (config: IDesignSystem): ComponentStyleSheet<IFlipperClassNameContract, IDesignSystem> => {
+/* tslint:enable:max-line-length */
+    const backgroundColor: string = get(config, "backgroundColor") || designSystemDefaults.backgroundColor;
+    const brandColor: string = get(config, "brandColor") || designSystemDefaults.brandColor;
+    const direction: Direction = get(config, "direction") || designSystemDefaults.direction;
+    const foregroundColor: string = get(config, "foregroundColor") || designSystemDefaults.foregroundColor;
+
+    return {
+        button: {
+            width: toPx(40),
+            height: toPx(40),
+            margin: "0",
+            border: `${toPx(1)} solid ${applyMixedColor(foregroundColor, backgroundColor, .86)}`,
```

Comment:
Just a thought - since it is not readily apparent what color this (or any of these applyMixedColor functions) outputs, I think we should leave a trailing comment that shows what hex or rgba colors is outputed

Reply:
Unless this is an established pattern already I suggest a new task for this.

Reply:
Yeah - not saying in this PR, but I think it would help in readability a good bit. Definitely another task

Reply:
I agree on this. Can you write up a task for this?

Reply:
Sure (#648)[https://github.com/Microsoft/fast-dna/issues/648]